scripts/upload_test_results/config.txt

```diff
@@ -0,0 +1,10 @@
+[Credentials]
```

Comment:
please add dummy information here, it should fail if this data is not provided correctly, we do not want people using the default and submitting in your name.